app/code/core/Mage/GoogleAnalytics/Block/Ga.php

```diff
@@ -196,6 +198,46 @@ protected function _getOrdersTrackingCodeUniversal()
         return implode("\n", $result);
     }
 
+    /**
+     * @return string|void
```

Comment:
```suggestion
     * @return string
```

as you do an echo in the template I would always return a string.

Reply:
true! thank you!